CIP-0068/README.md

```diff
@@ -0,0 +1,223 @@
+
+Lastly and most importantly, with this construction, the metadata can be used by a PlutusV2 script with the use of reference inputs [CIP-0031](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0031). This will drive further innovation in the token space. 
+
+## Specification
+
+This is the registered `asset_name_label` value
+
+| asset_name_label            | class        | description                                               |
+| --------------------------- | ------------ | --------------------------------------------------------- |
+| 100                         | NFT          | Reference NFT locked at a script containing the datum     |
+
+For a correct relationship between the `user token` and the `reference NFT` a few conditions **must** be met.
+- The `user token` and `reference NFT` **must** be under the same policy ID. 
+- For a specific `user token` there **must** exist exactly **one** `reference NFT`
+- The `user token` and associated `reference NFT` **must** follow the standard naming pattern. The asset name of both assets is prefixed with its respective `asset_name_label` followed by a pattern defined by the asset class (e.g. asset_name_label 222)
+
+Some remarks about the above,
+1. The `user token` and `reference NFT` do not need to be minted in the same transaction. The order of minting is also not important.
+2. It may be the case that there can be multiple `user tokens` referencing the same `reference NFT`. More explicit with this standard, if a specific `user token` has a quantity bigger than 1, then they all reference the same `reference NFT`.
+
+The datum in the output with the `reference NFT` contains the metadata at the first field of the constructor 0. The version number is at the second field of this constructor:
+```
+big_int = int / big_uint / big_nint
+big_uint = #6.2(bounded_bytes)
+big_nint = #6.3(bounded_bytes)
+
+metadata =
+  { * metadata => metadata }
+  / [ * metadata ]
+  / big_int
+  / bounded_bytes
+  
+version = big_int
+
+datum = #6.121([metadata, version])
+```
+
+## 222 NFT Standard
+
+Besides the necessary standard for the `reference NFT` we're introducing two specific token standards in this CIP. Note that the possibilities are endless here and more standards can be built on top of this CIP for FTs, other NFTs, semi fungible tokens, etc. The first is the `222` NFT standard with the registered `asset_name_label` prefix value
+
+| asset_name_label            | class        | description                                                          |
+| --------------------------- | ------------ | -------------------------------------------------------------------- |
+| 222                         | NFT          | NFT hold by the user's wallet making use of CIP-0025 inner structure |
```

Comment:
Shouldn't that:

- Be added to the CIP-0067's registry
- Be a separate proposal which piggybacks on this current proposal's structure? This would make the NFT standard more self-describing and have two immediate side-benefits (at least, obvious ones I see):
    - It'll make it more consistent for software down the lines to deal with such datum metadata. Right now, it feels that this asset class `222` is promoted to a blessed, implicit asset class, possibly omitted from the side registry. Having it part of the registry makes it more systematic to process by consumers down the line.
    - Moreover, this may keep the standard even more open as it would allow to define other kind of NFT standard (or simply, iterate on the existing one). It may help resolve some of the disputes regarding the adoption of this this particular NFT standard.  
 
?